docs/05-technical-reference/ac-01-application-gateway-details.md

```diff
@@ -52,4 +52,6 @@ In such a case, the `Location` header is modified so that the original target pa
 The modified `Location` header has the following format: `{APP_GATEWAY_URL}:{APP_GATEWAY_PORT}/{APP_NAME}/{SERVICE_NAME}/{SUB-PATH}`.
 
 This functionality makes the HTTP clients that originally called Application Gateway follow redirects through the Gateway, and not to the service directly. 
-This allows for passing authorization, custom headers, URL parameters, and the body without an issue.
\ No newline at end of file
+This allows for passing authorization, custom headers, URL parameters, and the body without an issue.
```

Comment:
Can you also copy the changes to https://github.com/kyma-project/application-connector-manager/blob/main/docs/user/technical-reference/07-10-application-gateway-details.md ?